plenty_api/api.py

```diff
@@ -1308,6 +1310,17 @@ def plenty_api_get_shipping_packages_for_order(self, order_id: int,
         return utils.summarize_shipment_packages(
             response=package_responses, mode=mode)
 
+    def plenty_api_get_amazon_product_types(self):
+        """
+        Get a list of all available Amazon product types.
+        https://developers.plentymarkets.com/en-gb/plentymarkets-rest-api/index.html#/Pim/get_rest_pim_amazon_product_types
```

Comment:
We don't need the URL here, it isn't all that valuable and also we already mention it in the documentation. So, please delete it from the function description.

Reply:
sure